scripts/dump_load_db.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
```

Comment:
Possibly run with `-e` to get automatic aborts if something does not work as expected?